backends/tfhe-cuda-backend/cuda/src/pbs/bootstrapping_key.cuh

```diff
@@ -116,12 +116,6 @@ void cuda_convert_lwe_programmable_bootstrap_key(cudaStream_t stream,
   switch (polynomial_size) {
   case 256:
     if (shared_memory_size <= cuda_get_max_shared_memory(0)) {
-      check_cuda_error(cudaFuncSetAttribute(
-          batch_NSMFFT<FFTDegree<AmortizedDegree<256>, ForwardFFT>, FULLSM>,
-          cudaFuncAttributeMaxDynamicSharedMemorySize, shared_memory_size));
-      check_cuda_error(cudaFuncSetCacheConfig(
```

Comment:
Maybe we could keep this one since it's always setting the same value `cudaFuncCachePreferShared`.

Reply:
Does it do anything actually? :thinking:

Reply:
Not sure. Probably none on Hopper, but could affect low end GPUs I think. Or maybe is a legacy thing that was only relevant on older archs. What if we bench this commit to see the impact if we remove it?

Reply:
Bench running: https://github.com/zama-ai/tfhe-rs/actions/runs/10995585669